src/Reader/Extension/PodcastIndex/Feed.php

```diff
@@ -224,12 +235,12 @@ public function getPodcastIndexUpdateFrequency(): object|null
     /**
      * Get the podcast people
      *
-     * @return list<stdClass|null>
+     * @psalm-return list<PersonObject>
      */
     public function getPodcastIndexPeople(): array
     {
         if (array_key_exists('people', $this->data)) {
-            /** @var list<stdClass|null> $people */
+            /** @var list<PersonObject> $people */
             $people = $this->data['updateFrequency'];
```

Comment:
Looks like this array key is incorrect - can we have a test to verify retrieval of People?

```suggestion
            $people = $this->data['people'];
```